docs/source/agent-framework/core-service-agents/volttron-central/volttron-central-platform.rst

```diff
@@ -1,23 +1,85 @@
 .. _VOLTTRON-Central-Platform-Agent:
 
 ===============================
-VOLTTRON Central Platform Agent
+Volttron Central Platform (VCP)
 ===============================
 
-The Platform Agent allows communication from a VOLTTRON Central instance.  Each VOLTTRON instance that is to be
-controlled through the VOLTTRON Central agent should have one and only one Platform Agent.  The Platform Agent must have
-the VIP identity of `platform.agent` which is specified by default by VOLTTRON
-:ref:`known identities <VIP-Known-Identities>`.
+The VCP exposes a VOLTTRON instance to a Volttron Central (VC) agent. The VC
+agent can either be on the same or a different VOLTTRON instance. The VCP agent will,
+once authenticated with the VC agent's instance, auto connect to the VC
+agent's instance and register itself on startup. The VCP instance will attempt
+to reconnect to the VC agent's instance if connection is disrupted. VCP has
+many configuration options available that can be set via the configuration store
```

Comment:
what are the many configuration options?

Reply:
Removed. Config shows options later in the document.

Reply:
Say that they will be discussed below...if you would.

Reply:
Updated